### PR TITLE
screen: new package at 4.8.0

### PR DESCRIPTION
**Pull request by sirtomato999**
Read: https://k1ss.org/style.txt
Read: https://k1ss.org/guidestones.txt

---

## Description of package
a basic fullscreen terminal multiplexer
## New package
- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/screen/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure --prefix="$1/usr"
+
+make
+make install
+
```
Comment (dylanaraps) [inline community/screen/build]: https://k1ss.org/style.txt 0400

**Comment by dylanaraps** [comment]
Also: https://k1ss.org/style.txt 1206

File: community/screen/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure \
+	--prefix=/usr
```
Comment (dylanaraps) [inline community/screen/build]: 4 spaces for indentation.

Comment (sirtomato999) [reply community/screen/build]: - [x]

File: community/screen/build (hunk 1 above)
Comment (sirtomato999) [reply community/screen/build]: ✔️

**Pull request merged**